cves/kernel/CVE-2022-28389.yml

```diff
@@ -278,8 +312,8 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: The mcba_usb driver uses IPC to communicate with the Linux kernel.
```

Comment:
This is not true, the driver is primarily called by the kernel via its `netdev_ops`. This is not related to IPC.